tests/test_compiler.py

```diff
@@ -400,13 +402,24 @@ def test_enrich_error_when_custom(compiler, project, owner, not_owner, connectio
     compiler.compile((project.contracts_folder / "HasError.sol",))
 
     # Deploy so Ape know about contract type.
-    contract = owner.deploy(project.HasError)
+    contract = owner.deploy(project.HasError, 1)
     with pytest.raises(contract.Unauthorized) as err:
         contract.withdraw(sender=not_owner)
 
     assert err.value.inputs == {"addr": not_owner.address, "counter": 123}
 
 
+@pytest.mark.skipif(APE_VERSION <= Version("0.6.11"), reason="Fails without fixes on main")
+def test_enrich_error_when_custom_in_constructor(compiler, project, owner, not_owner, connection):
+    contract_type = compiler.compile((project.contracts_folder / "HasError.sol",))[0]
+
+    # Deploy so Ape know about contract type.
+    with pytest.raises(contract_type.Unauthorized) as err:
```

Comment:
The test seems to be failing for me locally `AttributeError: 'ContractType' object has no attribute 'Unauthorized'`

Reply:
ok fixed now